cmd/test_load.go

```diff
@@ -0,0 +1,199 @@
+package cmd
+
+import (
+	"archive/tar"
+	"bytes"
+	"fmt"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"go.k6.io/k6/errext"
+	"go.k6.io/k6/errext/exitcodes"
+	"go.k6.io/k6/js"
+	"go.k6.io/k6/lib"
+	"go.k6.io/k6/lib/metrics"
+	"go.k6.io/k6/loader"
+)
+
+const (
+	testTypeJS      = "js"
+	testTypeArchive = "archive"
+)
+
+type loadedTest struct {
```

Comment:
My top-level package comment about naming would probably apply to this struct too 😄

Reply:
Resolved by https://github.com/grafana/k6/pull/2412/commits/6ffdb958f5e13bdc77fa7ef317af29c7be9b55a0